### DISCUSSION TITLE
4K Functionality

### DISCUSSION THREAD

**Discussion by jshq97** [sct/overseerr#490] [General]
Hey there,

Regarding the 4K Radarr/Sonarr settings for 'Default 4K' instance selection - What is the intended purpose for this?

I personally have 2 Radarr containers running (1 for 1080p, 1 for 4K) both mapped in Overseer, however I cannot see any option to select or prefer a 4K copy.

Is this something that should be working already, that I haven't configured correctly, or something that still needs to be implemented?

It looks like Overseer will be the perfect bridge for users to request a movie with both a 1080p & 4K copy being triggered without the need for an additional container such as Syncarr.

It would also be great for us to see which verison of plex content is already available & whether there are both 1080 & 4K versions already available within plex.

Love the work so far!!!

**Comment by sct** [comment]
Yeah the 4K functionality isn’t built yet outside of supporting multiple server configuration. It’s coming!

**Comment by sct** [comment]
This is now available on `develop` and will be in the next release.
